File: Tier1-Presentation/Domain/DTOs/RegisterPatientDto.cs
namespace Domain.DTOs;

public class RegisterPatientDto
{
    private string Name { get; set; }
    private string UserName { get; set; }
    private string Password { get; set; }

    public RegisterPatientDto(string name, string username, string password)
    {
        Name = name;
        UserName = username;
        Password = password;
    }
}

File: Tier2-BusinessLogic/src/main/java/via/sdj3/proofofconcept_v3/Dto/RegisterPatientDto.java
package via.sdj3.proofofconcept_v3.Dto;

import com.fasterxml.jackson.annotation.JsonProperty;

public class RegisterPatientDto {
    @JsonProperty("UserName")
    private String UserName;
    @JsonProperty("Password")
    private String Password;

    public RegisterPatientDto(String name, String password) {
        this.UserName = name;
        this.Password = password;
    }

    public RegisterPatientDto() {
    }

    public String getName() {
        return UserName;
    }

    public void setName(String name) {
        this.UserName = name;
    }

    public String getPassword() {
        return Password;
    }

    public void setPassword(String password) {
        this.Password = password;
    }
}
